src/data/markdown/docs/20 jslib/01 jslib/05 aws.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "aws"
+excerpt: "aws is a wrapper library around the native k6 http module"
```

Comment:
I would make the excerpt a bit more specific
`aws is a library implementing select APIs for accessing AWS services`

Reply:
Good catch, this was a leftover 👍🏻 
Rephrasing to `aws is a library implementing APIs for accessing a selection of AWS services`.